### ISSUE TITLE
VoiceOver doesn't read alerts values change quickly with long strings of aria-valuetext

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#140]
Originally reported in https://github.com/phetsims/gravity-force-lab-basics/issues/146, determined to be a general VoiceOver issue through this example HTML
<details>

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>title</title>
  </head>
  <body>

    <input type="range" id="range-1" aria-orientation="horizontal">
    <input type="range" id="range-2">

    <p id='live-element' aria-live="polite"></p>
  
  </body>

  <script type="text/javascript">
    const range1 = document.getElementById( 'range-1');
    const range2 = document.getElementById( 'range-2');
    const liveElement = document.getElementById( 'live-element' );

    let counter = 0;
    let timeoutId = null;
    const alertLater = ( alert ) => {
      if ( timeoutId !== null )
      window.clearTimeout( timeoutId );

      timeoutId = window.setTimeout( () => {
        liveElement.textContent = alert + ' ' + (++counter);
      }, 1000 );
    }

    let topCounter = 0;
    range1.addEventListener( 'change', ( event ) => {
      range1.setAttribute( 'aria-valuetext', 'a very long aria-valuetext string that will take time to read: ' + range1.value );
      alertLater( 'First Slider Changed' );

    } );

    range2.addEventListener( 'change', ( event ) => {
      range2.setAttribute( 'aria-valuetext', 'a very long aria-valuetext string that will take time to read: ' + range2.value );
      
      alertLater( 'Second Slider Changed' );  
  } );
  </script>
</html>
```
</details>

We may or may not spend time to find a project specific workaround, but either way we should submit a bug report to Apple.

**Comment by jessegreenberg** [comment]
In preparing the report to Apple I realized that this is a pretty complicated example HTML.

And when I remove the window.setTimeout and set the aria-live content immediately, the problem goes away.

**Comment by jessegreenberg** [comment]
From https://github.com/phetsims/gravity-force-lab-basics/issues/146

> So this isn't as straight forward as I thought, its more like there is a window of time where sending an alert to VoiceOver is bad. If set immediately on change, all is well. If set ~200 ms after, things are bad if the value is changed only once. If set ~1000 ms after, things are bad if the user changes the value more than once.

I guess the bug report for Apple would more generally be something like "Deferred alerts are often missing when using VoiceOver" or something.

**Comment by jessegreenberg** [comment]
I submitted a bug report on 7/12 for this.
